### ISSUE TITLE
VSCode shows same files with differents paths when working on SSH remote

### ISSUE THREAD

**Issue by zkelo** [microsoft/vscode#156602]
Does this issue occur when all extensions are disabled?: **I can't disable extensions because workspace needs one of it**

- VS Code Version: **1.69.2 (system setup)**
- OS Version: **Windows 11 10.0.22000 build 22000**

**Steps to Reproduce:**

1. Open remote folder through SSH;
2. Press F1 to open command palete;
3. Erase `>` char and type part of some filename.

When I typed part of filename VSCode suggest me same file with two different patches. First suggestion wih path of my workspace folder and second with remote path. For example: one suggestion is `workspace/database/migrations` and other is `/var/www/example.com/database/migrations`

![1659029594](https://user-images.githubusercontent.com/13527447/181602591-29c12200-e933-428f-9d68-285ef386b452.jpg)

**Comment by Drerootz1** [comment]
Repo pkg install

**Comment by zkelo** [comment]
> Repo pkg install

?

**Comment by roblourens** [comment]
I'm guessing that top section with the duplicates is the "Recently Opened" section. If that is the case, then the question is how and when those files got opened with a different URI.

Are you using a PHP extension? One possibility is that when you do a "go to definition" or similar command, the language server gives a URI that is formatted slightly differently, and that is where the duplicate comes from. Could be a different extension too. Can you look at the duplicates and try to remember whether there is something that you have done in all those files?

**Comment by dalmocoop** [comment]
Had the same issue in WSL2, code 1.69.1 with a couple of .tsx files. 

Something else I noticed was that, for those files listed with their full path, modifying them and closing the editor would not trigger a prompt to save/discard the changes, and git wouldn't pickup changes - but if I opened the file again the changes were there, as if closing the editor wasn't actually closing the file.

Reopening code did nothing, but reloading with extensions disabled fixed it. I've enabled extensions again and it seems alright so far.
